client/state/data-getters/index.js

```diff
@@ -23,6 +23,11 @@ import {
 import { convertToSnakeCase } from 'state/data-layer/utils';
 import { dummyTaxRate } from 'lib/tax'; // #tax-on-checkout-placeholder
 
+export const getAtURL = url =>
```

Comment:
Nit: according to our naming conventions, shouldn't this be named `getAtUrl`?

Reply:
I don't have strong feelings either way.

Reply:
I think we settled on `Url` and `Http` when used in `camelCase` at some point, mostly to avoid things like `getWPCOMHTTPURL()`.